pepper-apis/docs/specification/src/endpoints/studies.yml

```diff
@@ -6,9 +6,9 @@ get:
   description: returns a list of summary information for each of the studies under a given umbrella.
   parameters:
     - in: query
-      name: umbrella
+      umbrellaGuid: umbrella GUID
```

Comment:
We can leave the old query parameter (`name`) and prevent breaking the front-end. However, name containing a GUID sounds wrong. What's more important here - backward compatibility or correctness?

Reply:
Since this whole thing is effectively a breaking change, I'm inclined to say to just go ahead and rename the query parameter as well so we avoid potential confusion down the line.

By the way, the query parameter _name_ is not `name` but `umbrella`. Aka it's `name: <the_name>` and not `<the_name>: description`.

Reply:
I'd prefer we not rename this- I don't feel that adding `GUID` to the name contributes enough value to justify a breaking change. The query parameter is already named `umbrella` and that should be descriptive enough.

Reply:
Ok, so we have different opinions when it comes to renaming the query string param. On the one hand, renaming it will break the front-end. On the other, calling what is actually a GUID a "name" is quite confusing. Adding to this, the content of `umbrella.name` and `umbrella.guid` in the DB is the same now.

We need a consensus here.

Reply:
But it's not. The name of the query parameter was `umbrella`.

Reply:
Yup, just to reiterate what @bskinner said -- the query param name is `umbrella`. And either way is fine with me. If we're fine with keeping it as-is, then seems like less hassle to me. As the author of PR, what's your thoughts @raiks2?

Reply:
My vote would be to keep this as `umbrella` but change the logic and docs to clarify that this is the guid.